### PR TITLE
Chore/link to docs

### PR DESCRIPTION
**Pull request by mbush92**
Updated the README with a link to the official Docusaurus V2 docs now that they have been published.

https://v2.docusaurus.io/docs/next/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This WIP site will be used to present multilingual tutorial docs for RedwoodJS.
 
 ### Docusaurus 2 + Crowdin
 
-We are using a yet-to-be-documented release of Docusuraus 2 that includes integration with l10n management service Crowdin. While there are no docs to work off of yet, we have [this PR](https://github.com/facebook/docusaurus/pull/3325) and [this explanatory comment](https://github.com/facebook/docusaurus/issues/3317#issuecomment-742589241) to reference
+We are using a late stage Aplpha release of Docusuraus 2 that includes integration with l10n management service Crowdin. The docs for V2 of Docusaurus can be found [here](https://v2.docusaurus.io/docs/next/), we also have [this PR](https://github.com/facebook/docusaurus/pull/3325) and [this explanatory comment](https://github.com/facebook/docusaurus/issues/3317#issuecomment-742589241) to reference
```
Comment (clairefro) [inline README.md:20]:
```suggestion
We are using a late stage Alpha release of Docusuraus 2 that includes experimental integration with l10n management service Crowdin. The docs for V2 of Docusaurus can be found [here](https://v2.docusaurus.io/docs/next/), we also have [this PR](https://github.com/facebook/docusaurus/pull/3325) and [this explanatory comment](https://github.com/facebook/docusaurus/issues/3317#issuecomment-742589241) to reference
```

**Review by clairefro — CHANGES_REQUESTED** [review]
Thanks for the proactive edit. Actually the late-stage alpha release without documentation was meant to refer to the fact Docusaurus has not yet released documentation for i18n (the documentation is about to be released - a beta can be found in [this PR](https://github.com/facebook/docusaurus/pull/4014). Once published i18n docs will be included in the page you linked) 

But it is helpful to include the existing Docusuarus v2 docs so let's go ahead!

File: README.md (hunk 1 above)
Comment (mbush92) [reply README.md]: Thanks, I like that and have made the change.

**Comment by mbush92** [comment]
@clairefro when you get a chance, I made the requested changes and pushed them up.

**Review by clairefro — APPROVED** [review]

**Pull request merged**